functions/Get-DbaHelpIndex.ps1

```diff
@@ -0,0 +1,596 @@
+﻿function Get-DbaHelpIndex
+{
+<#
+.SYNOPSIS
+Returns size, row and configuration information for indexes in databases.
+
+.DESCRIPTION
+This function will return detailed information on indexes (and optionally statistics) for all indexes in a database, or a given index should one be passed along.
+As this uses SQL Server DMVs to access the data it will only work in 2005 and up (sorry folks still running SQL Server 2000).
+The data includes:
+	- ObjectName: the table containing the index
+	- IndexType: clustered/non-clustered/columnstore and whether the index is unique/primary key
+	- KeyColumns: the key columns of the index
+	- IncludeColumns: any include columns in the index
+	- FilterDefinition: any filter that may have been used in the index 
+	- DataCompression: row/page/none depending upon whether or not compression has been used
+	- IndexReads: the number of reads of the index since last restart or index rebuild
+	- IndexUpdates: the number of writes to the index since last restart or index rebuild
+	- SizeKB: the size the index in KB
+	- IndexRows: the number of the rows in the index (note filtered indexes will have fewer rows than exist in the table)
+	- IndexLookups: the number of lookups that have been performed (only applicable for the heap or clustered index)
+	- MostRecentlyUsed: when the index was most recently queried (default to 1900 for when never read)
+	- StatsSampleRows: the number of rows queried when the statistics were built/rebuilt
+	- StatsRowMods: the number of changes to the statistics since the last rebuild
+	- HistogramSteps: the number of steps in the statistics histogram
+	- StatsLastUpdated: when the statistics were last rebuilt
+
+.PARAMETER SqlServer
+SQLServer name or SMO object representing the SQL Server to connect to.
+
+.PARAMETER SqlCredential
+PSCredential object to connect as. If not specified, current Windows login will be used.
+
+.PARAMETER Databases
```

Comment:
It can accept multiple databases, and uses Get-ParamSqlDatabases which provides the databases parameter. Using database would require another helper function, or remove the potential for the drop down list.